sdk/servicefabric/azure-servicefabric/azure/servicefabric/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "8.0.0.0"
+VERSION = "8.2.0.0"
```

Comment:
@tjprescott @lmazuel if possible next time we bump this version could we remove the fourth part of the version? It breaks various assumptions when producing links and such in our release links. See https://dev.azure.com/azure-sdk/public/_build/results?buildId=1271182&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=036ed663-514a-59f3-0c88-23d2cf182147&l=76 or https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/2556. For this time I've hacked around it by pushing an extra release tag for azure-servicefabric_8.2.0.0 release tag.